src/routes/teams.rs

```diff
@@ -18,8 +21,136 @@ pub async fn team_members_get(
             user_id: data.user_id.into(),
             name: data.name,
             role: data.role,
+            permissions: data.permissions as u64
```

Comment:
This doesn't seem like it should be public, besides from what user is the owner

Reply:
Why not?

Reply:
There is no reason that anyone outside of the team should need or have access to a the specific permissions within the team.  That would be a security issue, since it would give people targets for which accounts to take over to get access to a mod, and it would give information about the structure of a team that shouldn't be publicly available.